docs/database/elasticsearch.md

```diff
@@ -64,7 +61,12 @@ HTTP method names are case-sensitive and `http.request.method` attribute value M
 Instrumentations for specific web frameworks that consider HTTP methods to be case insensitive, SHOULD populate a canonical equivalent.
 Tracing instrumentations that do so, MUST also set `http.request.method_original` to the original value.
 
-**[4]:** For network calls, URL usually has `scheme://host[:port][path][?query][#fragment]` format, where the fragment is not transmitted over HTTP, but if it is known, it should be included nevertheless.
+**[7]:** When observed from the client side, and when communicating through an intermediary, `server.address` SHOULD represent
+the server address behind any intermediaries (e.g. proxies) if it's available.
+
+**[8]:** When observed from the client side, and when communicating through an intermediary, `server.port` SHOULD represent the server port behind any intermediaries (e.g. proxies) if it's available.
+
+**[9]:** For network calls, URL usually has `scheme://host[:port][path][?query][#fragment]` format, where the fragment is not transmitted over HTTP, but if it is known, it should be included nevertheless.
```

Comment:
```suggestion
**[9]:** For network calls, the URL usually has `scheme://host[:port][path][?query][#fragment]` format, where the fragment is not transmitted over HTTP, If the scheme is known, it should be included.
```
Not sure if this is correct but original is way too long and ambiguous.

Reply:
"it" apparently refers to the fragment, not the scheme. To shorten the sentence, break it at "where":
```suggestion
**[9]:** For network calls, URL usually has `scheme://host[:port][path][?query][#fragment]` format. The fragment is not transmitted over HTTP, but if it is known, it should be included nevertheless.
```